### DISCUSSION TITLE
Update Docs for heroku_logs/logplex

### DISCUSSION THREAD

**Discussion by mms-gianni** [vectordotdev/vector#13957] [General]
I am ingesting logs from Heroku and struggled configuring it. 

The vector part is pretty good documented. But i missed the Heroku part. I took me a day to figure out how the drain should look like which i had to add with the Heroku-cli.

Here is what worked for me:
```
heroku drains:add "https://vector.mydomain.ch/events?application=myCustomAppTag" -a myApplication
```

But overall, still very happy with vector. Great work! 

PS: Since the docs are generated, i was not able to open a PR.

**Comment by jszwedko** [comment]
Thanks @mms-gianni !

If you like, you could add it the docs by adding a section like:

https://github.com/vectordotdev/vector/blob/36b9243b12a42f0160392b312693e57d81b0a56d/website/cue/reference/components/sources/aws_s3.cue#L162-L198

to

https://github.com/vectordotdev/vector/blob/master/website/cue/reference/components/sources/heroku_logs.cue
